Major documentation update

mliu49 opened the pull request:
This PR includes many documentation updates and fixes:

1. Updates theme to match RMG website
2. Fix a number of issues with docstrings and documentation pages that caused warnings during Sphinx processing
3. Change spelling of MolePro to Molpro everywhere (modules, classes, docstrings, comments)
4. Add a few new pages to the API documentation to reflect changes to RMG

Notes:
- The design was intentionally kept slightly different from the RMG website. One reason is that because the theme and general design is based on a Sphinx theme, it's not straightforward to mirror RMG website's style. Another reason is that slightly differentiating the designs might help people realize that they're separate websites.
- The number of warnings given by Sphinx has been reduced to 16, mainly duplicate labels or citations. I don't plan on following up on those in this PR unless someone else wants to do so.
- I'm not sure why MolePro was spelled that way, but based on their website, it should be Molpro. Since I added a API page on this module, I figured it would be best if we spelled it correctly.

Conflicting PR's:
- This conflicts with #1146 which modifies the MolePro module.
- This might conflict with #1183 which adds to the documentation.

It would be nice if this could be merged as soon as possible in order to update the documentation, but I wouldn't mind fast-tracking the conflicting PR's and then rebasing this on top.

mjohnson541 commented:
I checked it doesn't conflict with #1183.

mjohnson541 commented on documentation/source/_static/custom.css:
```diff
@@ -2,11 +2,51 @@
     border-right: none;
 }
+
+div.sphinxsidebarwrapper {
```
There are missing page things next to the links

mliu49 replied:
I'm not quite sure what you mean. We can also talk about this in person.

mliu49 replied:
Were you referring to the missing pictures? I just realized that the icons did not get committed because of the .gitignore file. It should be fixed now.

mliu49 commented on documentation/source/conf.py:
```diff
@@ -42,8 +42,10 @@
+year = datetime.datetime.now().year
 project = u'RMG Py'
-copyright = u'2002-2017, William H. Green, Richard H. West, and the RMG Team'
+copyright = u'2002-{0}, William H. Green, Richard H. West, and the RMG Team'.format(year)
```
The license is read automatically from `LICENSE.txt`. I created a separate pull request (#1277) updating that file and all the file headers.

I can look into whether or not it's possible to prevent Sphinx from automatically doing syntax highlighting.

mliu49 commented on rmgpy/solver/simple.pyx:
```diff
@@ -178,13 +178,18 @@ cdef class SimpleReactor(ReactionSystem):
-            Peff = P * y_specificCollider / sum(y)
-        if a secificCollider is mentioned
+
+        .. math:: P_{eff} = \\frac{P * y_{specificCollider}}{\\sum y}
```
Let's discuss this in person tomorrow if you're available.

mliu49 commented:
Could you do a final review of this PR and merge it if you think it's ready?

Updates since your review:
- Updated list of developers
- Disabled syntax highlighting for license
- Fixed the equation you mentioned
- Added missing images
- Added page for LiquidReactor in API
- Updated list of kinetics families and updated images

mjohnson541 commented on documentation/source/users/rmg/database/kinetics.rst:
```diff
@@ -211,234 +211,181 @@ Each reaction family contains the files:
-	:scale: 40% 
-
-
+There are currently 58 reaction families in RMG:
```
This table of images is giving me:  
System Message: ERROR/3 (/Users/mattjohnson/RMGCODE/RMG-Py/documentation/source/users/rmg/database/kinetics.rst, line 329)

Error in “table” directive: invalid option block.

mliu49 replied:
What version of docutils and sphinx do you have?

mjohnson541 commented:
Other than that issue I think everything else is ok.

mjohnson541 commented:
Ok, things look good now!
